bindings/core/src/method_handler/client.rs

```diff
@@ -239,9 +240,7 @@ pub(crate) async fn call_client_method_internal(client: &Client, method: ClientM
 
             Response::SignedTransaction(
                 (&block_builder
-                    .sign_transaction(PreparedTransactionData::try_from_dto_unverified(
-                        prepared_transaction_data,
-                    )?)
+                    .sign_transaction(PreparedTransactionData::try_from_dto(prepared_transaction_data)?)
```

Comment:
theres still 10 files with the normal try_from_dto, some with params. We should properly unify and have them all use the same trait

Reply:
I think @Alexandcoats had a valid reason not to do them, but anyway, we won't have time before the release.

Reply:
@thibault-martinez ??

Reply:
What